async-nats/examples/jetstream_pull.rs

```diff
@@ -33,13 +46,6 @@ async fn main() -> Result<(), async_nats::Error> {
         })
         .await?;
 
-    // Publish a few messages for the example.
```

Comment:
examples are meant to be runnable and working.
If we're not publishing, it will yield nothing.

Reply:
The thought here was to separate them to be run together just as pub and sub are. So based on this they could be recombined but then pub and sub examples should be also.

Reply:
Hmm, I got your point. Though I'm wondering how much value there is in separating the JetStream example.
The difference is that in Core NATS, it's at most once, and you need first to run `sub.rs` and then `pub.rs`. And they need to run simultaneously.

In Jetstream, you can Publish first and then get the messages anytime.
Hence it makes less sense to have them as separate files. You don't need to even create a consumer before publishing the messages. Just the Stream.

Reply:
Completely understandable. I will just rollback the split.